### ISSUE TITLE
BUG - [Critical] - [iOS] - VoiceOver user cannot send a secure message on iPhone

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va-mobile-app#11591]
## What happened?

Got this report from a blind Veteran yesterday. She cannot send a secure message in the app. Her workaround was to use VA.gov.

> Hi Martha, I am unable to send any secure messages. I select the facility, then it doesn’t let me put in who to send it to or the reason for the message, it did let me type in the subject line. Can you please help because I need to message my oncologist and this app is not working. Not sure if you guys know this or What is happening, this was not happening in the last two weeks..
> Not sure about last week, but I know two weeks ago it was working
> ...
> Yes ma’am, but this is different. This time it won’t allow me to even open the who I’m sending it to list, so I can’t pick anybody and I cannot pick what the messages about, so it won’t let me send it whatsoever. So the issue got worse instead of better.
> It would be equivalent to work stoppage, because I cannot send a message out to my provider

## Specs:

- Device: iPhone
- OS:
- User Account:
- Accessibility State:

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by digitalken** [comment]
@BeccaTupaj @artsymartha68 

Using the build that went to production (latest for users) I was not able to reproduce the problem of not being able to select a recipient for secure messaging.  I think this is the same bug that we have a fix going out for on 9/23.  I have made a recording of what happens.  When the user goes to try to select a recipient, the first time they double tap, only the keyboard is dismissed.  They try to double tap again, and the recipient is inserted in the "To:" field but there is no acknowledgement that it took place.  When going back to the field to re-read it with VoiceOver, the field is not read.  This could lead a non-sighted person to believe that the field is not filled, even though it is.  Please see the recording.

https://github.com/user-attachments/assets/69dd7988-19ae-4bbe-90f0-ddff39653f88


Here is the fix that is coming out on 9/23:


https://github.com/user-attachments/assets/8fdb664c-2772-42e6-9527-f76feb6851d6
